packages/light-apps/src/Content/Content.tsx

```diff
@@ -5,41 +5,44 @@
 import { Identity } from '@polkadot/identity-app';
 import { Transfer } from '@polkadot/transfer-app';
 import { Container } from '@polkadot/ui-components';
-
-import { inject } from 'mobx-react';
-import React from 'react';
-import { Route, RouteComponentProps, Switch } from 'react-router-dom';
+import keyring from '@polkadot/ui-keyring';
+import { inject, observer } from 'mobx-react';
+import * as React from 'react';
```

Comment:
why not `import React from 'react';` ?

Reply:
Ah, habit. In principle `import * as React from 'react';` is more correct, but since we added [allowSyntheticDefaultImports](https://github.com/polkadot-js/dev/blob/master/packages/dev/config/tsconfig.json#L45), TS is more lenient and allows this syntax.

Will change back for consistency

Reply:
ah interesting. thanks for the tip